registry.json

```diff
@@ -304,7 +306,7 @@
   "theseus": "npm:theseus",
   "theseus-angular": "npm:theseus-angular",
   "topo": "github:capaj/topo-browser",
-  "timers": "github:jspm/nodelibs-timers",
+  "timers": "github:jspm/nodelibs-timers",c
```

Comment:
Can you remove this character change? I think that may be invalidating the JSON.